mobile/src/main/java/org/openhab/habdroid/ui/activity/PageConnectionHolderFragment.kt

```diff
@@ -425,10 +425,15 @@ class PageConnectionHolderFragment : Fragment(), CoroutineScope {
                     }
                 }
```

Comment:
I think I'd rather do this here:

```
} else if (lastWidgetList != null) {
    // Either we didn't find the widget (possibly because the server didn't give us invisible widgets),
    // or we couldn't update it because we couldn't trust the data, so reload the page.
    // If we didn't have a widget list yet, simply ignore the event - we probably got it while loading the list.
    cancel()
    load()
}

Reply:
Almost, you just cut the last words of the comment ;-)

Reply:
😇